wireguard-vpn/js/MainController.js

```diff
@@ -300,6 +300,22 @@ Ext.define('Ung.apps.wireguard-vpn.MainController', {
         },function(ex){
             Util.handleException(ex);
         });
+    },
+
+    getNextUnusedPoolAddr: function() {
```

Comment:
Can this method be moved in Util.js and parameterize addressPool and store ?

Reply:
This is controller method and uses view-model of that specific view, so we can't move it to Util.js.

Reply:
Okay, fair enough!